bigframes/core/blocks.py

```diff
@@ -2474,7 +2474,7 @@ def column_ids(self) -> Sequence[str]:
         """Column(s) to use as row labels."""
         return self._block._index_columns
 
-    def to_pandas(self) -> pd.Index:
+    def to_pandas(self, *, sorted: Optional[bool] = None) -> pd.Index:
```

Comment:
I'm curious how this relates to the existing `ordered` bool on the public `to_pandas` method?

https://github.com/googleapis/python-bigquery-dataframes/blob/d5ae6807022aa556e80c76a254c2a72aef82f54b/bigframes/series.py#L326

and

https://github.com/googleapis/python-bigquery-dataframes/blob/d5ae6807022aa556e80c76a254c2a72aef82f54b/bigframes/dataframe.py#L1191

Should we align these names?

Reply:
Yes, the names mean the same thing. Renamed to `ordered`